parmed/openmm/parameters.py

```diff
@@ -531,7 +542,7 @@ def _write_omm_atom_types(self, xml_root, skip_types):
                 etree.SubElement(xml_section, 'Type', element=str(element), **properties)
 
     @needs_lxml
-    def _write_omm_residues(self, xml_root, skip_residues, valid_patches_for_residue=None):
+    def _write_omm_residues(self, xml_root, skip_residues, skip_duplicates=False, valid_patches_for_residue=None):
```

Comment:
`_write_omm_residues` is a private method.  No need to supply a default for `skip_duplicates` (especially not one that differs from the `write` default which is the only place this function is currently called).

I'd just make `skip_duplicates` a required argument, since intent should be explicit here.

Reply:
See new commit.